dev/docs.nix

```diff
@@ -23,5 +23,19 @@
         cd $files
         mkdocs build --strict --site-dir $out
       '';
+    docs-linkcheck = pkgs.testers.lycheeLinkCheck rec {
```

Comment:
Previously I hadn't been to concerned with implementing linkchecking but as this tester is in nixpkgs it make the maintenance burden of it very low so we may as well use it.